chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -0,0 +1,258 @@
+# ChainIndexer Documentation for RPC Providers
+
+# Table of Contents
+
+- [ChainIndexer Documentation for RPC Providers](#chainindexer-documentation-for-rpc-providers)
```

Comment:
Nit but I think this can be removed.  Assuming you're using markdown-all-in one, you can use `<!-- omit in toc -->`

https://markdown-all-in-one.github.io/docs/guide/table-of-contents.html#hiding-headings-from-toc

Reply:
Done.

Reply:
Removed TOC as discussed above.